app/controllers/api_controller/manager.rb

```diff
@@ -135,7 +135,7 @@ def update_multiple_collections(is_subcollection, target, type, resources)
         next if r.blank?
 
         rid = parse_id(r, type)
-        if rid && %w(create add).include?(action)
```

Comment:
@abellotti clash with action name `add` here - should we call it something else (and I'll put this back)?